src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -459,24 +452,55 @@ export const selectTotalCryptoBalanceWithDelegations = createSelector(
   },
 )
 
-export const selectPortfolioCryptoBalancesByAccountIdAboveThreshold = createDeepEqualOutputSelector(
-  selectAssets,
-  selectPortfolioAccountBalances,
-  selectPortfolioAssetBalances,
-  selectMarketData,
-  selectBalanceThreshold,
-  (_state: ReduxState, accountId?: string) => accountId,
-  (
-    assetsById,
-    accountBalances,
-    assetBalances,
-    marketData,
-    balanceThreshold,
-    accountId,
-  ): PortfolioBalancesById => {
-    const balances = (accountId ? accountBalances[accountId] : assetBalances) ?? {}
-    const aboveThresholdBalances = Object.entries(balances).reduce<PortfolioAssetBalances['byId']>(
-      (acc, [assetId, baseUnitBalance]) => {
+/**
+ * this selector is very specific; we need to consider
+ * - raw account balances, that are
+ * - above a threshold, including
+ *   - delegations
+ *   - redelegations
+ *   - undelegations
+ *   as delegations don't show in account balances, but we want them included in the total
+ */
+export const selectBalanceChartCryptoBalancesByAccountIdAboveThreshold =
+  createDeepEqualOutputSelector(
+    selectAssets,
+    selectPortfolioAccountBalances,
+    selectPortfolioAssetBalances,
+    selectMarketData,
+    selectBalanceThreshold,
+    selectPortfolioAccounts,
+    (_state: ReduxState, accountId?: string) => accountId,
+    (
+      assetsById,
+      accountBalances,
+      assetBalances,
+      marketData,
+      balanceThreshold,
+      portfolioAccounts,
+      accountId,
+    ): PortfolioBalancesById => {
+      const rawBalances = (accountId ? accountBalances[accountId] : assetBalances) ?? {}
+      // includes delegation, redelegation, and undelegation balances
+      const totalBalancesIncludingAllDelegationStates: PortfolioBalancesById = Object.entries(
+        portfolioAccounts,
+      ).reduce((acc, [accountSpecifier, account]) => {
+        Object.entries(account?.stakingDataByValidatorId ?? {}).forEach(
```

Comment:
`stakingDataByValidatorId` generalizes for all CosmosSDK-based chains that support the concept of staking to validators